cmd/commands/cobra.go

```diff
@@ -333,11 +333,11 @@ func Visit(cmd *cobra.Command, f func(c *cobra.Command) error) error {
 }
 
 func PrintSuccessfulCompletion(cmd *cobra.Command) {
-	fmt.Fprintf(cmd.OutOrStdout(), "\n%s completed successfully\n", cmd.CommandPath())
+	_, _ = fmt.Fprintf(cmd.OutOrStdout(), "\n%s completed successfully\n", cmd.CommandPath())
```

Comment:
What's producing the warning? Neither `go vet`, nor `golint` show anything on this line for me.

I'm not sure the visual noise of the blank identifier helps.

Reply:
Goland does. Here is an example:

<img width="771" alt="screenshot 2019-02-01 at 09 34 44" src="https://user-images.githubusercontent.com/445792/52111718-c6e2fc80-2604-11e9-804d-ac2c36f28553.png">

The blank identifier may be a visual noise but clearly states that the error is ignored, while one cannot make the difference between a statement that does not yield anything versus something that we ignore.